Package.swift

```diff
@@ -30,7 +30,10 @@ let package = Package(
             revision: "0c1de7149a39a9ff82d4db66234dec587b30a3ad"),
         .package(
             url: "https://github.com/apple/swift-syntax.git",
-            from: "509.0.0")
+            from: "509.0.0"),
+        .package(
+            url: "https://github.com/ordo-one/package-benchmark",
+            .upToNextMajor(from: "1.0.0"))
```

Comment:
I would recommend to add this as a dependency but follow the setup that we have done over in `swift-nio` and `swift-certificates` with a separate sub package.

Reply:
Fixed.

Reply:
So you should have a top level Benchmarks folder, with Benchmarks/Predicate under it, to follow their standard.

Reply:
Fixed.